couchbase/example/src/main/java/example/springdata/couchbase/repository/AirlineRepository.java

```diff
@@ -18,35 +18,31 @@
 import example.springdata.couchbase.model.Airline;
 
 import java.util.List;
-
-import org.springframework.data.couchbase.core.query.N1qlPrimaryIndexed;
-import org.springframework.data.couchbase.core.query.View;
-import org.springframework.data.couchbase.core.query.ViewIndexed;
-import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.couchbase.repository.CouchbaseRepository;
+import org.springframework.stereotype.Repository;
 
 /**
  * Repository interface to manage {@link Airline} instances.
  *
  * @author Chandana Kithalagama
  * @author Mark Paluch
+ * @author Denis Rosa
  */
-@N1qlPrimaryIndexed
-@ViewIndexed(designDoc = "airlines")
-public interface AirlineRepository extends CrudRepository<Airline, String> {
+@Repository
+public interface AirlineRepository extends CouchbaseRepository<Airline, String> {
```

Comment:
I think you should still extend the PagingAndSorting (or the crud) repository, not the couchbase one directly

Reply:
Fixed